module_auto_update/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Module Auto Update',
+    'summary': 'Automatically update Odoo modules',
+    'version': '10.0.1.0.0',
+    'category': 'Extra Tools',
+    'website': 'https://odoo-community.org/',
+    'author': 'LasLabs, '
+              'Juan José Scarafía, '
+              'Odoo Community Association (OCA)',
+    'license': 'LGPL-3',
+    'application': False,
+    'installable': True,
+    'post_init_hook': 'post_init_hook',
+    'external_dependencies': {
+        'python': [
+            'checksumdir',
+        ],
+    },
+    'data': [
+        'views/module_views.xml',
+        'data/cron_data.xml',
+    ],
+    'auto_install': True,
```

Comment:
I was just about to merge, but saw this. Please do not set it to autoinstall. Also, add dependency on `base` module.